### PR TITLE
Update date on the last note (advanced-communications.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6067.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-add-on-licensing/advanced-communications.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ With Advanced Communications, any meeting with more than 300 attendees will prov
 > The change from using Microsoft Stream to [OneDrive for Business and SharePoint for meeting recordings](../tmr-meeting-recording-change.md) will be a phased approach. At launch you'll be able to opt-in to this experience, in November you'll have to opt-out if you want to continue using Stream, and some time in early 2021 we'll require all customers to use OneDrive for Business and SharePoint for new meeting recordings.
 
 > [!NOTE]
-> Enterprise and Education SKUs can temporarily hold live events for up to 20,000 attendees, with an event duration of 16 hours per broadcast, and 50 concurrent events within a tenant until October 1, 2020.
+> Enterprise and Education SKUs can temporarily hold live events for up to 20,000 attendees, with an event duration of 16 hours per broadcast, and 50 concurrent events within a tenant until Jan 1, 2021.
```
Comment (JohanFreelancer9) [inline Teams/teams-add-on-licensing/advanced-communications.md:72]:
```suggestion
> Enterprise and Education SKUs can temporarily hold live events for up to 20,000 attendees, with an event duration of 16 hours per broadcast, and 50 concurrent events within a tenant until January 1, 2021.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**